modules/effects/spec/effects_metadata.spec.ts

```diff
@@ -36,17 +44,21 @@ describe('Effects metadata', () => {
         @Effect({ dispatch: false })
         e: any;
         f = createEffect(() => of({ type: 'f' }), { dispatch: false });
+        g = createEffect(() => of({ type: 'f' }), {
```

Comment:
To possibly avoid confusion later
```suggestion
        g = createEffect(() => of({ type: 'g' }), {
```